src/neon/intrinsics.rs

```diff
@@ -301,7 +407,7 @@ pub fn zerou8x16() -> uint8x16_t {
 
 #[inline]
 pub unsafe fn vpaddq_u8(a: uint8x16_t, b: uint8x16_t) -> uint8x16_t {
-    mem::transmute(vpaddq_u8_(mem::transmute(a), mem::transmute(b)))
+    mem::transmute(vaddq_u8_(mem::transmute(a), mem::transmute(b)))
```

Comment:
addq and paddq are different intrinsics! This will lead to wrong results

Reply:
ty!